### ISSUE TITLE
Standardize Tags - Merge Tags with the Same Meaning

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#961]
I'm fairly sure this has been done at least once in the past.
It's perhaps time to clean up the tags again, merging different wordings of the same idea.

For example,  these mean the same thing.
female speaker
said by female

The original use of "female" by itself had this meaning, too.
Perhaps it still does.

**Comment by PaulPeer** [comment]
Yes, two years ago we had a partial clean up, like "Flemish" to "Dutch - Belgium", but the "bad" tags always come back. It's difficult to persuade people to use the existing tags and not creating new ones.

**Comment by jiru** [comment]
Maybe Tatoeba should provide a tool to corpus maintainers to allow them to rename, merge etc. tags so that this task could be performed by non-developers more regularly.

**Comment by PaulPeer** [comment]
+1

**Comment by ckjpn** [comment]
To be on the safe side, you could limit this tool to admins for the initial testing period, just to make sure it's working properly first and to give us time to write up some guidelines.

**Comment by RyckRichards** [comment]
+1

**Comment by sacredceltic** [comment]
But if users are not informed that tags are being merged (which they might object to), merging them is useless. This must be publicised on Tatoeba's wall.
